.pre-commit-config.yaml

```diff
@@ -1,11 +1,16 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     sha: master
     hooks:
-    -   id: check-added-large-files
-        args: ['--maxkb=100']
-    -   id: check-case-conflict
-    -   id: end-of-file-fixer
-    -   id: fix-encoding-pragma
-    -   id: mixed-line-ending
-        args: [--fix=lf]
+      - id: check-added-large-files
+        args: [ '--maxkb=100' ]
+      - id: check-case-conflict
+      - id: end-of-file-fixer
+      - id: fix-encoding-pragma
```

Comment:
Relaying a comment from @pijyoi but this should likely be

```
    - id: fix-encoding-pragma
      args: ['--remove']
```

As we no longer support py2, so we should be nuking those, not adding more of them